kore/src/main/scala/org/kframework/definition/outer.scala

```diff
@@ -311,6 +359,45 @@ case class Module(val name: String, val imports: Set[Module], localSentences: Se
   override def equals(that: Any) = that match {
     case m: Module => m.name == name && m.sentences == sentences
   }
+
+  def toFlatModules() : (String, Set[FlatModule]) = {
+    var flatSelf = new FlatModule(name, importedModuleNames, sentences, att)
+    var flatModules = Set(flatSelf) ++
+      (imports.size match {
+        case 0 => Set()
+        case _ => imports.map(m => m.toFlatModules._2).flatten
+      })
+    (name, flatModules)
+  }
+
+}
+
+object FlatModule {
+  def apply(name: String, unresolvedLocalSentences: Set[Sentence]): FlatModule = {
+    new FlatModule(name, Set(), unresolvedLocalSentences, Att.empty)
+  }
+}
+
+case class FlatModule(name: String, imports: Set[String], localSentences: Set[Sentence], @(Nonnull@param) val att: Att = Att.empty)
+  extends OuterKORE with Sorting with Serializable {
+}
```

Comment:
Let's move these to `outer-ext.scala`.

Reply:
`object Import` as well?

Reply:
I think `object Import` is useful even for non-ext modules.

Reply:
So lets keep it here.